From 6a980bbf9648e3529d2722de4aa6efd58e0f6206 Mon Sep 17 00:00:00 2001
From: "Darryl L. Pierce" <mcpierce@apache.org>
Date: Tue, 14 Oct 2014 17:08:53 +0000
Subject: [PATCH 04/12] QPID-6150: Add __init__.py to setup.py for linear
 store.

The file was not being included in the source tarball.

git-svn-id: https://svn.apache.org/repos/asf/qpid/trunk@1631819 13f79535-47bb-0310-9956-ffa450edef68
---
 qpid/tools/setup.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/qpid/tools/setup.py b/qpid/tools/setup.py
index 5f6c4ec..e58270b 100755
--- a/qpid/tools/setup.py
+++ b/qpid/tools/setup.py
@@ -49,7 +49,8 @@ setup(name="qpid-tools",
                "src/py/qmf-tool"],
       data_files=[("libexec", ["src/py/qpid-qls-analyze"]),
                   ("share/qpid-tools/python/qlslibs",
-                   ["src/py/qlslibs/anal.py",
+                   ["src/py/qlslibs/__init__.py",
+                    "src/py/qlslibs/anal.py",
                     "src/py/qlslibs/efp.py",
                     "src/py/qlslibs/err.py",
                     "src/py/qlslibs/jrnl.py",
-- 
2.1.0

